.pre-commit-config.yaml

```diff
@@ -8,11 +8,11 @@ repos:
       - id: debug-statements
 
 
-  #- repo: https://github.com/PyCQA/flake8
-   # rev: 3.9.2
-    #hooks:
-     # - id: flake8
-      #  additional_dependencies: ["flake8-bandit", "flake8-bugbear"]
+  - repo: https://github.com/PyCQA/flake8
```

Comment:
Yep let's add this back :)

Reply:
Hmm...not commented out on mine, guess I didnt' add it back properly